### ISSUE TITLE
Can't claim rewards

### ISSUE THREAD

**Issue by cptblack1** [pbca26/hw-kmd-wallet#268]
I tried claiming the reward and it seem to get stuck. The wallet first showed the transaction as pending for over 5 mins and then the transaction disappeared from HW-KMD-Wallet (I closed and opened the wallet) and goes back to the state it was at before I tried claiming - the unclaimed rewards still accumulating and the wallet showing up to the last time I did a successful transaction, the pending one gone.

If I try claiming again everything seems fine and I get a green tick next to approve transaction but then the final stage shows:

Broadcasting transaction to the network

Error Missing inputs. Code:-25

I guess the wallet is still trying to used the original address but the coins have been moved to another address within the same wallet?

I removed and adding my ledger again and also tried changing the Explorer API end point, again without any joy.

I tried using Ledger Live and moved my coins to the previous address and that was successful but it's still showing my rewards unclaimed and no new transactions in HW-KMD-Wallet app.

Looking at my address on the explorer, it seems fine and my coins have moved to another address.

I tried all this on both the software app, as well as the web app and got the same problem. Using windows 11 and a couple of Chrome based browsers (Chrome, Brave and Avast).

**Comment by pbca26** [comment]
Can you retry now?

**Comment by cptblack1** [comment]
Hi 
I just tried, after scanning my ledger it showed the correct balance. 

I tried claiming my rewards and it showed the transaction as pending, even though the explorer showed 3 confirms by then. 

I closed and opened again and the pending transaction had vanished and it showed my starting balance with the rewards still showing as claimable. The explorer and ledger live show the right balance. 

So basically the same problem, the only difference being that my wallet has been reset to show from today rather than the 8th.

Thanks  :)
